src/components/calendar/__snapshots__/calendar.test.jsx.snap

```diff
@@ -107,6 +107,7 @@ exports[`<Calendar /> has defaults 1`] = `
               class="not-current-date"
               data-epoch="1593302400000"
               role="gridcell"
+              tabindex="-1"
```

Comment:
We need tab indices to populate the `relatedTarget` on Focus events (e.g. blur event)